package.json

```diff
@@ -331,6 +331,7 @@
     "redux-thunk": "^2.3.0",
     "throttleit": "^1.0.0",
     "webext-redux": "^2.1.4",
-    "webtorrent": "^0.114.0"
+    "webtorrent": "^0.114.0",
+    "web-discovery-project": "brave/humanweb#v1.1.5"
```

Comment:
Note: we'll also need to update the repo name here once it has been moved

Reply:
Also, does this reflect the last release?

Reply:
I've been building this against your branch. I'll change it here once the repo is opened up.